routes/api.php

```diff
@@ -87,7 +87,10 @@
 Route::name('v4_bible_books_all')->get('bibles/books/',                            'Bible\BooksController@index');
 Route::name('v4_bible.one')->get('bibles/{bible_id}',                              'Bible\BiblesController@show');
 Route::name('v4_bible.all')->get('bibles',                                         'Bible\BiblesController@index');
-Route::name('v4_bible.defaults')->get('bibles/defaults/types',                           'Bible\BiblesController@defaults');
+Route::name('v4_bible.defaults')->get('bibles/defaults/types',                     'Bible\BiblesController@defaults');
+Route::name('v4_bible.copyright')->get('bibles/{bible_id}/copyright',              'Bible\BiblesController@copyright');
+Route::name('v4_bible.chapter')
+        ->middleware('APIToken')->get('bibles/{bible_id}/chapter',                 'Bible\BiblesController@chapter');
```

Comment:
seems like (at least) BiblesController.php is missing from this PR?

Reply:
Yes Jon, is present: https://github.com/faithcomesbyhearing/dbp/pull/132/files#diff-d606e8933076ea05f0ae62f67ca49383